lib/components/ResourceList/ResourceList.tsx

```diff
@@ -0,0 +1,55 @@
+import { ListBase, ListItem } from '../List';
+import type { ListItemSize, ListProps, ListSpacing } from '../List';
+
+export interface ResourceListItemProps {
+  id: string;
+  ownerName: string;
+  resourceName: string;
+  ownerLogoUrl?: string;
+  ownerLogoUrlAlt?: string;
+  size?: ListItemSize;
+  controls?: React.ReactNode;
+}
+
+export interface ResourceList {
+  items: ResourceListItemProps[];
+  spacing: ListSpacing;
+}
+
+export const ResourceListItem = ({
```

Comment:
I think I would favor having ResourceListItem as a separate file in the ResourceList folder. It makes the files much cleaner to work.

 Having said that, is `ResourceListItem` is necessary at all since only uses ListItem directly?

Reply:
I see. If there is an intention of using `ResourceListItem ` separately, e.g. using without `ResourceList` then I agree, and also if we need to "mix" items.

Not sure if I understand, but what I meant was that `ResourceListItem ` could be exported in `ResourceListItem .tsx`, not be a part of `ResourceList.tsx`

Reply:
Nice. I'll move the ListItem to a separate file.

Reply:
@allinox Ah, yes! I agree. 

To conclude:

- Separate file but keep ResourceListItem as own component
- ResourceList uses ResourceListItem
- Future discussion: Whether **one** List component is the mother of all *ListItem?
- For now, we are fine with having a separate mother parent for each type, at least for ResourceList.

Am I getting close? :D

Reply:
Agree! 😄